masakari-controller/controller/masakari_controller.py

```diff
@@ -47,10 +46,14 @@
 import controller.masakari_starter as starter
 from controller.masakari_util import RecoveryControllerUtil as util
 from controller.masakari_util import RecoveryControllerUtilDb as util_db
+from controller.masakari_util import LogProcessBeginAndEnd
+from oslo_log import log as oslo_logging
 import controller.masakari_config as config
 import controller.masakari_worker as worker
 import db.api as dbapi
 
+log_process_begin_and_end = LogProcessBeginAndEnd()
+LOG = oslo_logging.getLogger('controller.masakari_controller')
 
```

Comment:
I appreciate your idea!
I will push patch for resolving it after other problems are fixed.